doc/dbus/index.html

```diff
@@ -1,6 +1,26 @@
+<!--
+echo "<ul>" ;\
```

Comment:
Um, so could we directly generate it from the `Makefile`?

Reply:
Quite likely :smile: , in the next PR